### PR TITLE
Fix workflows and checks

### PR DESCRIPTION
**Pull request by svartkanin**
In this PR
- Fix the broken github workflows 
- Update the translation check

### CODE REVIEW

**Comment by svartkanin** [comment]
@Torxed the new flow still doesn't show up... anything that needs to be enabled?

File: .github/workflows/flake8.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ jobs:
         steps:
             - uses: actions/checkout@v3
             - run: pacman --noconfirm -Syu python python-pip
+            - run: python -m venv .venv
+            - run: source .venv/bin/activate
```
[the remark this answers is not in the document]
Comment (svartkanin) [reply .github/workflows/flake8.yaml]: right, thanks for that
I think adding the flag `--break-system-packages` should do the trick

**Comment by svartkanin** [comment]
@Torxed can we merge this PR meanwhile to enable the checks again otherwise all other PRs go through without validation

**Comment by Torxed** [comment]
Ah yea, `--break-system-packages` was recently required because Arch adopted the "mark python packages as externally managed" which is a great feature to remind people about the dangers of pip install vs pacman :D

**Comment by svartkanin** [comment]
@Torxed is there a way to better enforce merge blocks for the future if checks are failing

**Comment by Torxed** [comment]
There are, the repo can configure that all checks have to pass.
The down side is that if you're on the road and can't fix them - but need to merge stuff so other PR's can get it that solves issues (but have dependencies) you're forced wait.

So it's down to me making sure I don't merge stuff that breaks checks - but some times I do just because we need a PR in quickly (but fails flake8 for instance).

**Pull request merged**